lib/functionTemplates/java8/binaris.yml

```diff
@@ -3,3 +3,4 @@ functions:
         file: build/libs/example.jar
         entrypoint: ExampleFunction
         runtime: java8
+        executionModel: exclusive
```

Comment:
this is the default, why do we need to specify it?

Reply:
I think it add clarity.

Reply:
what do you understand when you look at it? (pretend that you only know java)

Reply:
I know that I can google `executionModel: exclusive` and understand what is going on (and that there are other options).

Reply:
are you sure you can google it?